util/file_util.py

```diff
@@ -629,6 +643,9 @@ def file_write_csv_dict(py_dict: dict,
       is used as the key's column name. If no columns are specified for values,
       column names are picked from each entry's value if the value is a dict.
       Else the value is written as column name 'value'.
+    key_column_name: name of the column used as key.
+      if None, the first column is used as key.
```

Comment:
Will it be more intuitive to use None to indicate key is ignored?

Reply:
Good suggestion. Done